.mergify.yml

```diff
@@ -0,0 +1,80 @@
+# This file was automatically generated by sbt-typelevel-mergify using the
+# mergifyGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the mergify configuration
+# to meet your needs, then regenerate this file.
+
+pull_request_rules:
+- name: merge scala-steward's PRs
+  conditions:
+  - author=scala-steward
```

Comment:
Oh, I forgot that we need some additional configuration to override this PR author setting. Could you please add [these sbt settings](https://github.com/typelevel/natchez-http4s/blob/83ca4d833545066046f104eb24a3977d029eecff/build.sbt#L19-L24) to this project's `build.sbt`. Once this is done, you can confirm that the [generated YAML config looks similar to this](https://github.com/typelevel/natchez-http4s/blob/83ca4d833545066046f104eb24a3977d029eecff/.mergify.yml#L10-L14).